### DISCUSSION TITLE
Does it support anonymous login?

### DISCUSSION THREAD

**Discussion by mayone-du** [clerk/javascript#3542] [Support request - DO NOT OPEN HERE]
Hello. I'm wondering if there is support for anonymous login, similar to what is offered in Supabase, or if there are plans to support it in the future, or perhaps methods to implement it myself?

In the app I'm developing, I've implemented authentication using email addresses and email verification codes. I'm curious if there's a way to implement anonymous login in such a scenario.

**Comment by jescalan** [comment]
Not at the moment, but you're welcome to upvote/leave feedback on the feature here! https://feedback.clerk.com/roadmap/cc6fe926-0858-43c2-a362-e5f91e5081c3
